pwa-devdocs/src/peregrine/routing/index.md

```diff
@@ -9,141 +9,76 @@ Every URL request fetches new HTML from the server and the browser loads the ent
 This approach is inefficient because the same assets get loaded every time site navigation occurs.
 
 For a single-page application (SPA), such as a progressive web app, routing is performed on the client side.
-Single-page applications do not reload the browser during internal navigation.
+Single-page applications do not reload the browser for internal links.
```

Comment:
"Navigation" includes the forward and back buttons as well as form-driven navigations. "Internal links" seems less accurate to me.